src/fts-backend-xapian-functions.cpp

```diff
@@ -452,7 +449,7 @@ static void fts_backend_xapian_oldbox(struct xapian_fts_backend *backend)
 		}
         	/* End Performance calculator*/
 	
-                i_info("FTS Xapian: Done indexing '%s' (%ld msgs in %ld ms, rate: %.1f)",backend->oldbox,backend->perf_nb,dt,r);
+                i_debug("FTS Xapian: Done indexing '%s' (%ld msgs in %ld ms, rate: %.1f)",backend->oldbox,backend->perf_nb,dt,r);
```

Comment:
Not sure this is adequate, becuase if one need to put dovecot in debug mode, then all the debug messages comes, and fts debug gets lost. ALso performance data is quite usefull not in debug mode.